docs/source/model-registry.rst

```diff
@@ -230,6 +230,21 @@ token = <your Databricks shard access token>
     mlflow.set_registry_uri("databricks-uc://my-databricks-shard1")
 
 
+OSS Unity Catalog Model Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use an `OSS Unity Catalog <https://www.unitycatalog.io/>` server as your MLflow model registry, specify the UC server address in the MLflow registry URI in the format ``"uc:http://localhost:8080"``.  If your unity catalog server is configured to use authentication, set the environmental variable ``"MLFLOW_UC_OSS_TOKEN"``.
```

Comment:
This doesn't appear to hyper link `OSS Unity Catalog`.  Any idea what the issue here is?

Reply:
/ai What's missing?

Reply:
```suggestion
To use an `OSS Unity Catalog <https://www.unitycatalog.io/>`_ server as your MLflow model registry, specify the UC server address in the MLflow registry URI in the format ``"uc:http://localhost:8080"``.  If your unity catalog server is configured to use authentication, set the environmental variable ``"MLFLOW_UC_OSS_TOKEN"``.
```

`_` is missing.